src/run.ts

```diff
@@ -13,6 +15,11 @@ export async function run() {
     if (eventPayload.issue.state_reason !== "completed") {
       return logger.info("Issue was not closed as completed. Skipping.").logMessage.raw;
     }
+    if (!(await preCheck())) {
+      const result = logger.error("All linked pull requests must be closed to generate rewards.");
+      await githubCommentModuleInstance.postComment(result.logMessage.diff);
+      return result.logMessage.raw;
```

Comment:
I thought our logger abstracts all this away

Reply:
Because we wanted to separate Supabase from the logger for a few reasons:
- it would require every project to link to Octokit
- that would force every project to include Octokit which differs in use between Actions and Workers
- some do no need to log them (mostly actions) as the logs are saved within the Action run (and now within worker with the new Logs)

My suggestion was to have optional packages that you can add on init, like Octokit does if you want to use pagination for example, where we could link a package that would post comments.

Reply:
I think you can file a spec before it's forgotten about

Reply:
There was a task carried out about this I remember, let me find it.